README.md

```diff
@@ -0,0 +1,5 @@
+Introductory to Python with `@Lily`
+-----------------------------------
+
+1. Create Rest Endpoint
+2. Integrate to persistent data
```

Comment:
I think it's better if we mention which feature is available and which one is still in development

Reply:
Change to table 🙇